fineract-provider/src/main/java/org/apache/fineract/infrastructure/core/exception/MultiException.java

```diff
@@ -87,7 +86,6 @@ public void printStackTrace(PrintStream s) {
         int i = 0;
         for (Throwable e : throwables) {
             s.print(++i + ".");
-            e.printStackTrace(s);
```

Comment:
@thesmallstar this needs to be kept. It's "special", and SupressWarning is OK here (only)